rules/S7042/cfamily/metadata.json

```
@@ -0,0 +1,24 @@
+{
+  "title": "\"std::views::as_const\" should be used to iterate on writable ranges without modifying them",
```

I would prefer something like?
```suggestion
  "title": "\"std::views::as_const\" should be used to prevent modification of the range elements",
```

---

90a03217c92a02f3079d1f7c0bc6115830caab4f